### PR TITLE
Add retrospectively events in 2021

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding LCI and McGill events.

cc @jburel

### CODE REVIEW

File: events/index.html [hunk 1]
```diff
@@ -10,7 +10,19 @@
         <hr class="whitespace">
         <div id="community-posts" class="row">
             <div class="small-12 medium-8 medium-offset-2 columns">
-                <h6><i class="fa fa-star event-future"></i> 27-29 January, 2021</h6>
+                <h6><i class="fa fa-calendar"></i> February 18-19, 2021</h6>
+                <h2><a href="omero-users-workshop-montreal-online-february-2021.html">OMERO Workshop, McGill University, Montreal</a></h2>
+                <p class="card-caption">A 2 day workshop on how to use OMERO held remotely.</p>
```
Comment (jburel) [inline events/index.html:15]: 2-day

File: events/omero-users-workshop-montreal-online-february-2021.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: held remotely
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Date: 18-19th November 2021</h2>
```
Comment (jburel) [inline events/omero-users-workshop-montreal-online-february-2021.html:20]: February

File: events/omero-users-workshop-montreal-online-february-2021.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: approx 4 hours per day</h2>
```
Comment (jburel) [inline events/omero-users-workshop-montreal-online-february-2021.html:26]: drop ``approx``

File: events/omero-users-workshop-montreal-online-february-2021.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                <p class="event-day-description">The Open Microscopy Environment (OME) is an open-source software project that develops tools that enable access, analysis, visualization, sharing and publication of biological image data. We support more than 150 image data formats across many imaging modalities including medical imaging, high-content screening and whole-slide imaging.
+                We support and visualize multi-channel 3D timelapse images.
+ 
+                OMERO, a component of OME, is a software platform for image data management and analysis which will be presented in this course. 
+                Import, organisation, viewing, searching, annotation and publishing of imaging data with OMERO will be shown.
```
Comment (jburel) [inline events/omero-users-workshop-montreal-online-february-2021.html:31]: Analysis using Fiji was also shown

File: events/workshops-lci-karolinska-february-2021.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+---          
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organisers: <a href="https://ki.se/en/bionut/lci-microscopy-course" target="_blank">LCI at Karolinska Institutet</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: Stockholm, but OMERO.figure workshop was held remotely
```
Comment (jburel) [inline events/workshops-lci-karolinska-february-2021.html:15]: drop ``but``

**Comment by joshmoore** [comment]
Not necessarily for this PR but what are your thoughts on starting to link to the downloads pages from each of these?

**Comment by jburel** [comment]
We could point to the downloads page but the version used during the workshops might not matched the version available at https://www.openmicroscopy.org/omero/downloads/ (if this what you mean)

**Comment by pwalczysko** [comment]
> We could point to the downloads page but the version used during the workshops might not matched the version available at https://www.openmicroscopy.org/omero/downloads/ (if this what you mean)

I think what @joshmoore means is https://github.com/ome/www.openmicroscopy.org/pull/449/commits/d5bdfb30629112cbb8faed13c35105d53c26619e - this can be added to the workshops which are in the past, but typically, the materials simply do not exist at the time when the posts in Events on www are made. We can add these retrospectively as in https://github.com/ome/www.openmicroscopy.org/pull/449/commits/d5bdfb30629112cbb8faed13c35105d53c26619e

**Comment by joshmoore** [comment]
:+1: for adding retrospectively, or in the future, proactively saying, "all resources related to this course will be available under:"

**Pull request merged**